### DISCUSSION TITLE
Contributo ai Comuni per l'ingresso in ANSC

### DISCUSSION THREAD

**Discussion by SAlberto** [italia/ansc#1372] [Announcements]
#390 
Buongiorno,
avremmo bisogno di sapere la **data ultima** entro la quale tutti i comuni italiani dovranno **obbligatoriamente** essere subentrati in ANSC.

In attesa di un cortese riscontro
Ringrazio anticipatamente
Distinti saluti

**Comment by AlessandriniD** [comment]
Buongiorno,
la data ultima entro la quale tutti i comuni italiani dovranno obbligatoriamente entrare in ANSC non è stata ancora stabilita. Infatti in base all'art. 13 c. 4 del DM del Ministero dell'Interno del 18 ottobre 2022:
"I comuni aderiscono all'ANSC  entro  diciotto  mesi  dalla  data della comunicazione da parte del Ministero dell'interno della messa a disposizione del servizio in ambito nazionale". 
Quindi, da quando il Ministero comunicherà la data di apertura a livello nazionale di ANSC, i comuni avranno 18 mesi di tempo per aderire. Grazie e buona giornata.
